src/components/recovery/RecoveryInfo/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { SvgIcon, Tooltip } from '@mui/material'
+import type { ReactElement } from 'react'
+
+import WarningIcon from '@/public/images/notifications/warning.svg'
+
+export const RecoveryInfo = (): ReactElement => {
+  return (
+    <Tooltip title="This recovery attempt was not created via the interface" placement="top" arrow>
```

Comment:
> This recovery attempt was not created via the interface

That it wasn't created via the interface is a poor way to say it's a fraudulent transaction.

Design-wise, I wouldn't even display them as recovery attempts. Too easy to confuse with a legit recovery.

<img width="802" alt="Screenshot 2023-11-21 at 11 09 13" src="https://github.com/safe-global/safe-wallet-web/assets/381895/feadaec4-c226-4029-9379-f82a2b861d41">

Reply:
I've renamed the transaction type of "Malicious transaction" (coloured in `error.main`) and changed the tooltip to "Suspicious activity" in 6896ea6497e6318c8ffb226c27737f612c12f456.

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/97d8a9da-a1f2-41eb-8ffe-4e36eaa77fa5)